app/controllers/api/v1/people_controller.rb

```diff
@@ -47,6 +43,12 @@ def court_cases
         end
       end
 
+      def timetable
+        timetable = People::RetrieveTimetable.call(person)
```

Comment:
this is a bit inconsistent: `People::RetrieveCourtCases.call()` returns a `response` object with `court_cases` as a property; whereas `People::RetrieveTimetable.call()` returns the actual timetables. 

Probably best to update `People::RetrieveTimetable.call()` to return a response object like `People::RetrieveCourtCases.call()`, then it can better handle Nomis errors

Reply:
Yeah, I actually needed that to handle errors - see subsequent commits :)